webpack.config.shared.js

```diff
@@ -12,8 +12,8 @@ module.exports = {
     module: {
       rules: [
         {
-          test: /\.(svg|ttf|woff|woff2|eot|gif)$/,
-          use: "asset/inline",
+          test: /\.(png|jpg|svg|ttf|woff|woff2|eot|gif)$/,
```

Comment:
@arslanashraf7 would you shed light on these two file types? as these will be handled by Webpack to avoiding unexpected behavior.
we have something similar in mit-open https://github.com/mitodl/mit-open/blob/main/frontends/mit-learn/webpack.config.js#L179

Reply:
I have verified locally. All the assets including PNG/JPG files are loading correctly. Thanks @mudassir-hafeez and @arslanashraf7 for looking into this. Merging this PR.